server/fleet/orbit.go

```diff
@@ -25,6 +25,10 @@ type OrbitConfigNotifications struct {
 	// host.IsEligibleForWindowsMDMUnenrollment for the list of conditions).
 	NeedsProgrammaticWindowsMDMUnenrollment bool `json:"needs_programmatic_windows_mdm_unenrollment,omitempty"`
 
+	// EnforceBitLocker is sent as true if Windows MDM is
+	// enabled and the device should encrypt its disk volumes with BitLocker.
+	EnforceBitLocker bool `json:"enforce_bitlocker_encryption,omitempty"`
```

Comment:
cool! I will copy and paste this in my PR.

should we make this consistent? either both the json value and the struct field should be `EnforceBitLocker` + `enforce_bitlocker` or `EnforceBitLockerEncryption` + `enforce_bitlocker_encryption`

Reply:
Let's go with `EnforceBitLockerEncryption` + `enforce_bitlocker_encryption` as it clearly defines the purposes + we might need to implement bitlocker decryption notification in the future

Reply:
Fixed [here](https://github.com/fleetdm/fleet/pull/14022/files#diff-926109db4eb6bfb52416300b7a363636f98be23d7a9447956546d73c2f10fe7eR35)